marklogic-data-hub/src/main/java/com/marklogic/hub/impl/DataHubImpl.java

```diff
@@ -859,7 +859,10 @@ public boolean upgradeHub(List<String> updatedFlows) throws CantUpgradeException
                 text = Pattern.compile("^(\\s*)id\\s+['\"]com.marklogic.ml-data-hub['\"]\\s+version.+$", Pattern.MULTILINE).matcher(text).replaceAll("$1id 'com.marklogic.ml-data-hub' version '" + version + "'");
                 text = Pattern.compile("^(\\s*)compile.+marklogic-data-hub.+$", Pattern.MULTILINE).matcher(text).replaceAll("$1compile 'com.marklogic:marklogic-data-hub:" + version + "'");
                 FileUtils.writeStringToFile(buildGradle, text);
-                hubConfig.getHubSecurityDir().resolve("roles").resolve("flow-operator.json").toFile().delete();
+                hubConfig.getHubSecurityDir().resolve("roles").resolve("data-hub-role.json").toFile().delete();
+                hubConfig.getHubSecurityDir().resolve("roles").resolve("hub-admin-role.json").toFile().delete();
+                hubConfig.getHubSecurityDir().resolve("users").resolve("hub-admin-user.json").toFile().delete();
+                hubConfig.getHubSecurityDir().resolve("users").resolve("data-hub-user.json").toFile().delete();
             }
```

Comment:
@aebadirad I dont think we should delete them. We are altering the .old file that way and user will lose what they had pre upgrade. Collecting the obsolete files and not moving them over seems to be the right approach. What do you think? Co hubInit is going to create the new roles and users.

Reply:
Fair enough. Then let's add them under hub-internal-config obsolete files (all 4)

Reply:
Looks like we only had data-hub-user and data-hub-role in 3.0
hub-admin-role and hub-admin-user probably came in 4.0+. Will test by marking them obsolete for 4.x to 4.3 upgrade.